AWSSDKSwiftCLI/Tests/AWSSDKSwiftCLITests/Commands/PrepareReleaseTests.swift

```diff
@@ -45,15 +45,7 @@ class PrepareReleaseTests: CLITestCase {
         createPackageVersion(previousVersion)
         createNextPackageVersion(newVersion)
 
-        let buildRequest = """
-        {
-            "features": []
-        }
-        """
-        FileManager.default.createFile(atPath: "../build-request.json", contents: Data(buildRequest.utf8))
-
-        let mapping = "{}"
-        FileManager.default.createFile(atPath: "../feature-service-id.json", contents: Data(mapping.utf8))
+        createBuildRequestAndMapping()
```

Comment:
Creation of JSON files extracted to a reusable helper below, so it can be used in the "no code changes" test case below